lib/storage/metadata/MetadataWrapper.js

```diff
@@ -515,6 +517,75 @@ class MetadataWrapper {
                 return cb();
             });
     }
+
+    /**
+     * Put bucket indexes
+     *
+     * indexSpec format:
+     * [
+     *      { key: Map<key, value>, name: <id 1>, ... , < backend options> },
+     *      ...
+     *      { key: Map<key, value>, name: <id n>, ... },
+     * ]
+     *
+     *
+     * @param {String} bucketName bucket name
+     * @param {Array<Object>} indexSpecs index specification
+     * @param {Object} log logger
+     * @param {Function} cb callback
+     * @return {undefined}
+     */
+    putBucketIndexes(bucketName, indexSpecs, log, cb) {
+        log.debug('put bucket indexes');
+
+        if (!supportsIndexingMDBackends.has(this.implName)) {
+            log.debug('error from metadata', { implName: this.implName });
```

Comment:
Wouldn't this better be an error log? (unless it's meant to be called with incompatible backends in normal circumstances). Also logging the error type can be useful:

```suggestion
            log.error('error from metadata', { error: 'NotImplemented', implName: this.implName });
```